src/Service/Core.php

```diff
@@ -129,6 +128,36 @@ public function createNodes($module) {
           ]
         )
       );
+
+      // Translate the node.
+      $languages = $this->languageManager->getLanguages();
+      foreach ($languages as $langcode => $language) {
+        if (!$language->isDefault()) {
+          // Get the JSON config for this language.
+          $jsonTranslation = $this->getConfigFromJson($module, 'config', $langcode);
+
+          // If the config is different from the default, add a translation.
+          if ($jsonTranslation != $json) {
+            $translation = $node->addTranslation($langcode);
+            $translation->set('title', $title);
```

Comment:
Is it worth combining all these translation sets into an array of values and a foreach?

Reply:
I think probably not - we only want to create the translation if the config is different

what would we gain by that approach? in my mind it would just make it a bit harder to read

Reply:
Would look a bit cleaner Imo, as it means we only have a single translation->set then we run our values through it. I guess it depends on whether we expect to expand on this in the future, if not probably not worth changing it,